src/Http/Http/test/Features/ResponseBodyPipeFeatureTests.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.IO.Pipelines;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Http.Features
+{
+    public class ResponseBodyPipeFeatureTests
+    {
+        [Fact]
+        public void ResponseBodyReturnsStreamPipeReader()
+        {
+            var context = new DefaultHttpContext();
+            var expectedStream = new MemoryStream();
+            context.Response.Body = expectedStream;
+
+            var provider = new ResponseBodyPipeFeature(context);
+
+            var pipeBody = provider.ResponseBodyPipe;
+
+            Assert.True(pipeBody is StreamPipeWriter);
```

Comment:
Same as reading size. Write some data into the pipe and assert it ended up in the stream.